tests/queries/0_stateless/02995_new_settings_history.sh

```diff
@@ -17,7 +17,8 @@ $CLICKHOUSE_LOCAL --query "
     new_settings AS
     (
         -- Ignore settings that depend on the machine config (max_threads and similar)
-        SELECT name, default FROM system.settings WHERE default NOT LIKE '%auto(%'
+        -- Ignore settings that, for historic reasons, have different values in Cloud
+        SELECT name, default FROM system.settings WHERE default NOT LIKE '%auto(%' AND name NOT IN ('max_table_size_to_drop', 'max_partition_size_to_drop')
```

Comment:
Where do `max_table_size_to_drop` and `max_partition_size_to_drop`? Where do they cause issues?

Reply:
They cause issues when enabling this test in private

Reply:
I'm going to rework this a bit to know exactly about CLICKHOUSE_CLOUD